primitives/src/block.rs

```diff
@@ -180,7 +180,7 @@ pub struct Header {
 impl Header {
     /// The number of bytes that the block header contributes to the size of a block.
     // Serialized length of fields (version, prev_blockhash, merkle_root, time, bits, nonce)
-    pub const SIZE: usize = 4 + 32 + 32 + 4 + 4 + 4; // 80
+    pub const SIZE: usize = 80; // 4 + 32 + 32 + 4 + 4 + 4
```

Comment:
Honestly, this looks like a bug in `cargo mutants`. Having an expression is the entire point - to not have to compute it. Though the magic number "80" is pretty well-known, so I don't mind it here, but I'd prefer to not do such things in the future.

Reply:
Yeah, the other option to remove the mutant would have been to check the value in a test.  I just figured the two different ways of writing it above are basically the same.

Reply:
FTR I'm not fussed, either with or without the change is the same to me.

Reply:
> Yeah, the other option to remove the mutant would have been to check the value in a test.

Oh I meant a more meaningful test, call a function that includes some calculation that uses `SIZE` (assuming we have one).

Reply:
The most meaningful test would be `assert_eq!(header.consensus_encode(sink).unwrap(), Header::SIZE);` but I'd argue that it actually checks the encoding impl rather than `SIZE` even though `cargo mutants` disagrees. :)

Reply:
I reverted the change and added test that manually calculates the size of a `Header` and compares it to `Header::SIZE`